site/docs/presto.md

```diff
@@ -17,4 +17,4 @@
 
 # Presto
 
-Iceberg connector is part of Presto SQL's master branch, although support for many features is still under development. Current status of the connector is detailed in the [presto-iceberg README](https://github.com/prestosql/presto/blob/master/presto-iceberg/README.md).
+Iceberg connector is part of Presto SQL's master branch, and feature development is complete.  Connector documentation can be found in [Presto Iceberg Connector](https://prestosql.io/docs/current/connector/iceberg.html).
```

Comment:
How about rewording it a bit to something like this:

```
The Iceberg connector is available with Presto SQL. It is feature complete and usage details can be found in the [documentation](https://prestosql.io/docs/current/connector/iceberg.html).
```

Reply:
Indeed that's better; updated and force-pushed.